.github/workflows/scripts/common/name-artifacts.sh

```diff
@@ -21,7 +21,7 @@ NAME=""
 
 if [ "${OS}" == "macos" ]; then
   NAME="PCSX2-${OS}-${GUI_FRAMEWORK}"
-elif [[ ("${OS}" == "windows" && "$BUILD_SYSTEM" != "cmake") || ("$OS" == "linux" && "$GUI_FRAMEWORK" == "QT") ]]; then
+elif [[ ("${OS}" == "windows" && "$BUILD_SYSTEM" != "cmake") ]]; then
   NAME="PCSX2-${OS}-${GUI_FRAMEWORK}-${ARCH}-${SIMD}"
```

Comment:
Does the change no include windows?  It seems like there are still traces of explicitly defining `simd` in a few places:
https://github.com/TellowKrinkle/pcsx2/blob/c357e64dea294952185e58f24a9b2bc7a5d2f1b4/.github/workflows/windows_build_qt.yml#L20-L23
I would have expected this could all be cleaned up.

Reply:
There aren't any in the release pipeline, but MSBuild builds don't use Multi-ISA, and we'll be keeping at least some of them around to ensure you can continue to use MSBuild to build, even if we don't use it for published releases.

(Windows release pipeline builds were changed to use cmake)